kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/food2door/orders/OrderRetriever.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.good.patterns.challenges.food2door.orders;
+
+import com.kodilla.good.patterns.challenges.food2door.shops.GlutenFreeShop;
+import com.kodilla.good.patterns.challenges.food2door.shops.HealthyShop;
+import com.kodilla.good.patterns.challenges.food2door.data.Product;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class OrderRetriever {
+    public List<OrderRequester> retrieve() {
+        List<OrderRequester> requests = new LinkedList<>();
+        HealthyShop healthy = new HealthyShop();
+        GlutenFreeShop glutenFree = new GlutenFreeShop();
```

Comment:
Referencje do shop'ow powinny byc trzymane w OrderProcessor. To OrderProcessor powien rozpoznac ktory shop powinien byc uzyty przy kazdym requescie